package.json

```diff
@@ -27,10 +27,12 @@
     "internal:release:next": "yarn prerelease && yarn changeset publish --tag next",
     "serve": "cd ./site && next",
     "start": "npm-run-all --parallel --print-label watch serve",
-    "test": "mocha --require ./config/babel/register.cjs './packages/*/test/**/*.{js,ts}'",
+    "test": "run-s test:mocha test:jest",
```

Comment:
Also, why are we using `run-s test:mocha test:jest` instead of `test:mocha && test:jest`?

Reply:
Just committed the change you propose.